yaml/service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: minio-kv-svc
+spec:
+  type: LoadBalancer
```

Comment:
We would probably use this project for internal use as a `ClusterIP` or `NodePort`, it might be expensive to expose with an LB, instead of routing through an existing Ingress